Client/src/components/details/details.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import io from 'socket.io-client';
+
+
+function details() {
+
+
+  const [data, setData] = useState({});
+
+  useEffect(() => {
+    const socket = io('http://localhost:4000', {
```

Comment:
Instead of having URL here store it in a variable and user here, this way when your are using the same URL in multiple component you can make changes from one file, rather than having to change in different files.

Reply:
Noted!